src/cordial_face/launch/face_server.launch

```diff
@@ -3,6 +3,7 @@
     <arg name="web_directory_path" value="$(find cordial_face)/web"/>
     <arg name="port" default="8080"/>
     <arg name="is_launch_websocket" default="true"/>
+    <rosparam file="$(find cordial_face)/../topic_parameters/topic_parameters.yaml" />
```

Comment:
Instead of using find the relative package, with respect to the current package, let's just put `topic_parameters.yaml` into the `cordial` metapackage.